elasticsearch/connection/http_urllib3.py

```diff
@@ -62,33 +76,37 @@ def __init__(self, host='localhost', port=9200, http_auth=None,
                 self.headers[k.lower()] = headers[k]
 
         self.headers.setdefault('content-type', 'application/json')
-        ca_certs = CA_CERTS if ca_certs is None else ca_certs
         pool_class = urllib3.HTTPConnectionPool
         kw = {}
         if use_ssl:
+            if not ca_certs and not ssl_context and verify_certs:
```

Comment:
`ca_certs` have a default value so it's ok if it's empty. This could work if we haven't removed line 65